android/app/build.gradle

```diff
@@ -37,14 +37,12 @@ dependencies {
     implementation project(':capacitor-cordova-android-plugins')
 
     // Import the BoM for the Firebase platform
-    implementation platform('com.google.firebase:firebase-bom:29.3.0')
+    implementation platform('com.google.firebase:firebase-bom:32.7.2')
 
     // Declare the dependencies for the Crashlytics and Analytics libraries
     // When using the BoM, you don't specify versions in Firebase library dependencies
     implementation 'com.google.firebase:firebase-crashlytics'
     implementation 'com.google.firebase:firebase-perf'
-    // https://github.com/mixpanel/mixpanel-android/issues/744
-    implementation("com.google.firebase:firebase-iid")
```

Comment:
`firebase-iid` is deprecated as of `firebase-bom` v31.0.0, see [release notes](https://firebase.google.com/support/release-notes/android#bom_v31-0-0). As far as I can tell, we don't currently use mixpanel, so it seems that we can just remove this declaration?

Reply:
Judging by the previous comment we have never used mixpanel but had to include the the package because of an error being triggered. Hopefully should work now without (don't see anything in the issue updates but will try a quick test locally)